### PR TITLE
Remote work & communication guidelines

### PR DESCRIPTION
**Pull request by krantzinator**
- addresses #33 and has information that could be shared with #53 
- @blacknight811 & I will be incorporating these guidelines into our workflow over the next week, and will report back with any recommended changes

### CODE REVIEW

File: docs/Team/Working-Remote.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: "Tallwave Guidelines — Team — Working Remote"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# Our remote environment
+
+Tallwave as an organization is primarily on-site, but we have a handful of remote employees, as well as a strong culture of working from home when you need to go heads-down on a problem. Currently, the dev team is ~20% full-time remote. Given that, async communication is very important, for which we rely on Basecamp. Email is used mainly for company announcements and project-client communication, and video calls are for anything from client meetings to 1-on-1s to pair programming.
```
Comment (swilliams) [inline docs/Team/Working-Remote.md]: Specify that the 20% number is FTEs. Also mention that the majority of our contractors are remote and so it is important to have solid remote protocols.

File: docs/Team/Working-Remote.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+As an organization, we are working to make remote more integrated.
+
+Things we currently do are:
+- ensure company meetings have good audio for online attendees
+- regular 1-on-1 communication with same-department coworkers for pairing on non-client coding projects
+- fly remote employees to the Phoenix office roughly once/quarter for 3-5 days, covering the flight, hotel, and meals (and try to schedule those trips at the same time for the remotees!)
+- send gift cards to remote employees when they miss out on a project team's celebration
```
Comment (swilliams) [inline docs/Team/Working-Remote.md]: Cranky grammarian comment: For consistency, we've been using asterisks for unordered lists, and capitalizing the first word on a bullet point, and also ending them with periods. 😁

File: docs/Team/Working-Remote.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Things we want to do are:
+- make Virtual Coffees part of the culture, so an on-site employee can grab a remote coworker, or vice-versa, from any department for a chat every so often
+
+Things we'd like to do:
+- integrate better screensharing and pair-programming tools
+- make the inclusion of a GTM or Google Meet the default when creating meetings
```
Comment (swilliams) [inline docs/Team/Working-Remote.md]: We're moving away from GTM to Zoom... Maybe rephrase as the generic "video conference software"?

File: docs/Team/Working-Remote.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- have a shared microphone in larger company meetings that can be passed around, or have a culture where questions that are asked from the room are repeated into the central mic for the benefit of remote attendees
```
Comment (swilliams) [inline docs/Team/Working-Remote.md]: I dunno about this one... The majority of our meetings don't have a ton of people in one room. It's only for all-hands that this is an issue (I think?). The A/V issues also get complex... I'd file this as a nice-to-have but realistically wouldn't make a move on it for a while longer.

File: docs/Team/Working-Remote.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+## General Communication
+
+- Set your notification preferences
```
Comment (swilliams) [inline docs/Team/Working-Remote.md]: This goes beyond remote, imo. Everyone in office or not needs to be mindful about their notifications... Consider moving it to its own page altogether.

Comment (krantzinator) [reply docs/Team/Working-Remote.md]: @swilliams it's own page or to the Operations page?

File: docs/Team/Working-Remote.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# Our remote environment
+
+Tallwave as an organization is primarily on-site, but we have a handful of remote employees, as well as a strong culture of working from home when you need to go heads-down on a problem. Currently, ~20% of the full-time dev team and the majority of our contractors are remote. Therefore, async communication is very important, for which we rely on Basecamp. Email is used mainly for company announcements and project-client communication, and video calls are for anything from client meetings to 1-on-1s to pair programming.
+
+As an organization, we are working to make remote more integrated.
+
+Things we currently do are:
+* ensure company meetings have good audio for online attendees
```
Comment (swilliams) [inline docs/Team/Working-Remote.md]: Capitalize bullet sentences and end with periods. 😃

Comment (krantzinator) [reply docs/Team/Working-Remote.md]: @swilliams does that include the "header" bullets, like `* Set your notification preferences`?

Comment (swilliams) [reply docs/Team/Working-Remote.md]: If they're headers they should be header tags (and then title-cased). Like `## Set Your Notification Preferences`.

**Comment by krantzinator** [comment]
@TinaHeiligers @swilliams I moved a couple of things to a `Communication` page, and added a new paragraph there as well titled `### Alert your team if you usually work outside of 9-to-5 hours`.

**Comment by swilliams** [comment]
Just realized that the `Communication` page is missing. @krantzinator can you add it lickety-split? 😄

**Comment by krantzinator** [comment]
@swilliams doh I pushed it to the wrong branch. PR incoming.

File: docs/Team/Working-Remote.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+* Fly remote employees to the Phoenix office roughly once/quarter for 3-5 days, covering the flight, hotel, and meals (and try to schedule those trips at the same time for the remotees!).
+* Send gift cards to remote employees when they miss out on a project team's celebration.
+
+Things we want to do are:
+* Make Virtual Coffees part of the culture, so an on-site employee can grab a remote coworker, or vice-versa, from any department for a chat every so often.
+* Integrate better screensharing and pair-programming tools.
+* Make the inclusion of video conferencing the default when creating meetings.
+
+Nice-to-haves as we grow:
+* For all-hands meetings or company standups, have a shared microphone that can be passed around, or have a culture where questions that are asked from the room are repeated into the central mic for the benefit of remote attendees.
```
Comment (vinnieking06) [inline docs/Team/Working-Remote.md]: I like this.
